### ISSUE TITLE
How to make the candidates file?

### ISSUE THREAD

**Issue by parulj3795** [devanshkv/fetch#5]
Hi. I am following your paper 'Towards deeper neural networks for Fast Radio Burst detection'. Could you please direct on how to make the candidate file? The information in README.md is not very clear to me.

**Comment by devanshkv** [comment]
To create a candidate.csv file, you would need

1. The path filterbank file
2. Signal to noise ratio of the candidate
3. Start time (all single pulse search pipeline lines like `heimdall` or `singlepulsesearch.py` Give the start time of the candidate in the filterbank file). In case of `heimdall` its the third column.
4. DM of the candidate
5. Box car width, this is samples as a power of two. So if you boxcar width is 8 samples, this number is `int(log2(8)) = 3`.
6. Label is not required for prediction and can be ignored. If you are planning to train the classifiers then the label is 0 for RFI and 1 for FRBs.
7. Kill mask. This is a simple text file with channel numbers in the filterbank which you would like to mask. This can also be ignored if there are no bad channels.

Stitch this into a .csv file which looks like:
`/path/to/filterbank/myfilterbank.fil,S/N,start_time,dm,boxcar_width`


Hope this helps, let me know if you have more questions. If you have a specific candidate file from a pipeline for which you want help, please go ahead and paste it below.

**Comment by parulj3795** [comment]
In order to train my network, I need to have a training data set. Can I create fake FRB candidates using candmaker.py ?

**Comment by vandana55** [comment]
Hi Devansh
Could you please elaborate how many total features you have considered while generating candidate file?
following are few features
1. Signal to noise ratio of the candidate
2. DM of the candidate
3. Box car width
is there any other feature considered while designing candidate file.

**Comment by vandana55** [comment]
With respect to above query..
I think these are the Parameters / features used...
List of candidate parameters (fil_name, snr, width, dm, label, tcand(s))

**Comment by devanshkv** [comment]
@parulj3795 `candmaker.py` cannot make fake FRBs. You can try using [burst_search](https://github.com/kiyo-masui/burst_search) or [simpulse](https://github.com/kmsmith137/simpulse).

**Comment by devanshkv** [comment]
@vandana55 do you mean the `candidate.csv` for training FETCH? For that see Table 2 and Figure 4 in our [paper](https://arxiv.org/pdf/1902.06343.pdf).
